webpack.config.ts

```diff
@@ -1,12 +1,14 @@
-'use strict';
-const path = require('path');
+import path from 'path';
+import CopyWebpackPlugin from 'copy-webpack-plugin';
+import TerserPlugin from 'terser-webpack-plugin';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+import webpack from 'webpack';
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports,@typescript-eslint/no-var-requires
 const SizePlugin = require('size-plugin');
-const CopyWebpackPlugin = require('copy-webpack-plugin');
-const TerserPlugin = require('terser-webpack-plugin');
-const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
-module.exports = (env, argv) => ({
-	devtool: 'sourcemap',
+module.exports = (_env: string, argv: Record<string, boolean|number|string>): webpack.Configuration => ({
+	devtool: 'source-map',
```

Comment:
The types for webpack complained about this, so I put it to what it was expecting.